packages/runner-commons/src/assets/i18n/web/en/en.json

```diff
@@ -54,6 +54,16 @@
   "description": "Press specified key multiple times using | as: num|{key}",
   "wording": "I press {int} times on {string}"
  },
+ {
+  "key": "key.when.keyboard.nextElement",
+  "description": "moves to the next html element that can be reached with Tab",
+  "wording": "I go to next keyboard element"
+ },
+ {
+  "key": "key.when.keyboard.previousElement",
+  "description": "moves to the next html element that can be reached with back Tab",
```

Comment:
C'est pas mieux de mettre to the previous élément dans la description ? Je pense que les 2 se disent.

Reply:
Rôle supprimé dans les fichiers packages/runner-commons/src/assets/i18n/web/en/en-roles.ts et packages/runner-commons/src/assets/i18n/web/fr/fr-roles.ts